### DISCUSSION TITLE
Support for MongoDB

### DISCUSSION THREAD

**Discussion by alexbevi** [wasp-lang/wasp#2104] [Archived]
One of the benefits of choosing the Prisma ORM is the variety of databases it supports, which is called out in Wasp's [blog post](https://wasp-lang.dev/blog/2022/11/28/why-we-chose-prisma) about this choice. Currently it seems only SQLite and Postgres are supported [backends](https://wasp-lang.dev/docs/data-model/backends) so what might the process look like to introduce additional databases (assuming they're currently supported by Prisma)?

I'd be interested in getting support for MongoDB added, however I'm not really sure how best to approach this. I know Prisma lists [MongoDB as a supported database|https://www.prisma.io/docs/orm/overview/databases/mongodb] so if I wanted to use this with my Wasp project what might a path forward look like?

**Comment by Martinsos** [comment]
@alexbevi While Prisma does support MongoDb, Wasp doesn't yet at the moment. It wouldn't be too difficult to add it, but we are trying to keep the number of variables lower while developing Wasp towards Beta. Once we hit 1.0 and current features reach maturity, we will be looking to expand more, including support for MongoDb!
